docs/assets/js/charts.js

```diff
@@ -63,7 +63,7 @@ const timeSeriesChartDefaults =
                 time:
                 {
                     format: 'YYYY-MM-DD',
-                    tooltipFormat: 'D MMMM YYYY',
+                    tooltipFormat: 'D MMM YYYY',
```

Comment:
Where is `format` used? I wonder if the formats should match...

Reply:
The `format` is the default tooltip configuration if you haven’t set a tooltip callback.

Having it spelled out here is mostly a defensive measure now that we have a callback. But I agree that it’s not really necessary anymore. Depends mostly on whether we can reuse that default configuration at a later point in time or not.